sanic_openapi/swagger.py

```diff
@@ -14,14 +14,7 @@
 dir_path = os.path.dirname(os.path.realpath(__file__))
 dir_path = os.path.abspath(dir_path + '/ui')
 
-
-# Redirect "/swagger" to "/swagger/"
-@blueprint.route('', strict_slashes=True)
-def index(request):
-    return redirect("{}/".format(blueprint.url_prefix))
```

Comment:
The redirect part still be removed. Maybe you can try reset `sanic_openapi/swagger.py` to get it back.
```shell
git checkout HEAD -- sanic_openapi/swagger.py
```